### PR TITLE
Added inputs to model class and defined construct_param_dict function

### PR DESCRIPTION
**Pull request by synapsidfan**
Established the input parameters for the model class and defined the function to create a dictionary of parameters.

### CODE REVIEW

File: pkmodel/model.py [hunk 1]
```diff
@@ -2,6 +2,9 @@
 # Model class
 #
 
+# from numpy import NaN
+
+
```
Comment (sscivier) [inline pkmodel/model.py:7]: Are these lines needed now?

File: pkmodel/model.py [hunk 2]
```diff
@@ -12,6 +15,33 @@ class Model:
...
+    def __init__(self, V_c, CL, K_a=None, Q_p1=None, Q_p2=None, V_p1=None,
+                 V_p2=None):
+        self.V_c = V_c
+        self.CL = CL
+        self.K_a = K_a
+        self.Q_p1 = Q_p1
+        self.Q_p2 = Q_p2
+        self.V_p1 = V_p1
+        self.V_p2 = V_p2
+        self.opt_params = [K_a, Q_p1, Q_p2, V_p1, V_p2]
```
Comment (sscivier) [inline pkmodel/model.py:27]: Duplication between the parameters in the list and not in the list.

File: pkmodel/model.py [hunk 3]
```diff
@@ -12,6 +15,33 @@ class Model:
...
+        for param in self.opt_params:
+            if param is not None:
+                param_dict[optional_param_name_list[i]] = param
+                i += 1
+    # def rhs(t, y, Q_p1, V_c, V_p1, CL, X):
+    #     q_c, q_p1 = y
+    #     transition = Q_p1 * (q_c / V_c - q_p1 / V_p1)
+    #     dqc_dt = dose(t, X) - q_c / V_c * CL - transition
+    #     dqp1_dt = transition
+    #     return [dqc_dt, dqp1_dt]
```
Comment (sscivier) [inline pkmodel/model.py:47]: Remove?

**Review by sscivier — CHANGES_REQUESTED** [review]
Dictionary generation looks good. Some cleaning up needed. Add testing to make sure the class and methods are working as you expect. Unittests currently failing because they haven't been updated.

**Review by Adrian-Mag — APPROVED** [review]

**Review by sscivier — APPROVED** [review]
One comment re adding a little info to docstrings but otherwise good

**Pull request merged**